app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -7,6 +7,7 @@ import { DialogContent } from '../dialog'
 import { Row } from '../lib/row'
 import { merge } from '../../lib/merge'
 import { caseInsensitiveCompare } from '../../lib/compare'
+import { canCreatePrivateRepo } from './publish-private-repository-checker'
```

Comment:
I couldn't think of a good name for this.  Any suggestions?

Reply:
`onPaidPlan`?

Reply:
I'm fine with the name you've chosen.